### PR TITLE
tests/utils: move 3 functions to their only user

### PR DESCRIPTION
**Pull request by dankenigsberg**
Nobody uses these functions outside config-tests, so nobody should be bothered by them when reading/importing tests/utils.go.

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [x] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [x] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [x] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [x] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [x] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by ShellyKa13 — COMMENTED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -206,7 +206,7 @@ format:
 fmt: format
 
 lint:
-	if [ $$(wc -l < tests/utils.go) -gt 2813 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
+	if [ $$(wc -l < tests/utils.go) -gt 2350 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
```
Comment (fossedihelm) [inline Makefile:209]: Can I ask you how is this value calculated? Thanks

Comment (ShellyKa13) [reply Makefile:209]: Its simply the number of lines in tests/utils.go after the move of the functions

Comment (fossedihelm) [reply Makefile:209]: :man_facepalming: I was looking at the previous version file! That's why something wasn't right! Thanks, sorry for the noise

**Review by EdDev — APPROVED** [review]

**Pull request merged**